### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by manuelmazzuola** [ramirezd42/ng2-flex-layout#3]

**Comment by backlit-studio** [comment]
Yeah, sorry about that. I'll update the readme and/or take the project down.

I'm no longer continuing development on this project in favor of just pulling in the sass files required for the css-only implementation that angular material 1 provides. Check out this thread for more info: https://github.com/angular/material2/issues/946
